src/MaterialDesignThemes.Wpf/TextFieldAssist.cs

```diff
@@ -8,6 +8,15 @@ namespace MaterialDesignThemes.Wpf;
 /// </summary>
 public static class TextFieldAssist
 {
+    /// <summary>
+    /// Used by text field "wrappers" (i.e. controls hosting a text field and decorating on top of it) to signal to the text field that the mouse is over it,
+    /// when in fact it is over a sibling (i.e. something in the wrapper) which is visually placed on top of the text field.
+    /// </summary>
+    public static readonly DependencyProperty IsMouseOverProperty = DependencyProperty.RegisterAttached(
```

Comment:
I think this approach looks great. Though I am thinking we probably want to put this into a different class that is inside of the Internal namespace (but have the class be public) so that people don't assume this is a simply customization like the other APs in here.

Reply:
@Keboo Fair point. Although I don't see an existing class that seems fit for purpose. If I create a new one, what should the naming be? It would be confusing to have another `TextFieldAssist` simply in another namespace...

![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/19572699/2c0a5b02-f08b-4339-980b-2997f802ac33)